engine/apps/slack/alert_group_slack_service.py

```diff
@@ -34,41 +30,12 @@ def __init__(
         else:
             self._slack_client = SlackClient(slack_team_identity)
 
-    def update_alert_group_slack_message(self, alert_group: "AlertGroup") -> None:
```

Comment:
decided to move all of this logic to [the new `update_alert_group_slack_message` Celery task](https://github.com/grafana/oncall/pull/5287/files#diff-cc25198aab50b80e59957f5b6c0f495503fcc7b5459bc66ad47baf5db8364504R35)